std_msgs/Header header
LineSegment[] line_segments
